### ISSUE TITLE
Joint ER[P/F] and topo plot

### ISSUE THREAD

**Issue by jona-sassenhagen** [mne-tools/mne-python#2664]
Once #2662 is merged, only one aspect is missing for my favorite way to plot.

![4bcd9962-73d1-11e5-9b66-6e601899d9fb](https://cloud.githubusercontent.com/assets/4321826/11531433/1f8ef4a8-98fd-11e5-9e42-218d07e26f4b.png)

Combined time series and topoplots, like EEGLAB has.

I'd like to implement this next, but I'm wondering about the API. A function `mne.viz.evoked.joint_plot`? Another arg for `evoked.plot`, or maybe `evoked.plot_topomap`? A new method (`joint_plot`) for evoked?

**Comment by agramfort** [comment]
I would add times param to evoked.plot to add topos from specific time
instants.

**Comment by jona-sassenhagen** [comment]
I'd also want to expose most or all other params for plot_topoplot though, and also allow auto times (e.g. the new 'peaks'  functionality), how about a `list | dict` arg? (if list, treat it as times)

**Comment by agramfort** [comment]
hum then can you make it work just by passing a fig or axis object that
contains the evoked.plot to a new function. Our plot functions have already
too many parameters.

**Comment by jona-sassenhagen** [comment]
For the future: EEGLAB also has joint time-frequency/topo plots.
![bildschirmfoto 2015-12-02 um 14 17 47](https://cloud.githubusercontent.com/assets/4321826/11531789/cabe82e2-98ff-11e5-8254-e797f473ee7e.png)

So if we also want that (IMO they're very nice), one should think about an API that also covers that case.

**Comment by agramfort** [comment]
nilearn.plotting handles this by returning more than a fig and having an
object that allows to interact with plot and complement it.

Something along these lines:

fig = evoked.plot()
fig.add_topo(times=...)

**Comment by jona-sassenhagen** [comment]
Hm ... but that's not really MNE style is it? Also I'm not sure I could program that ...
Both the topoplot and the ts plot need access to the original evoked object. And API wise, I think a method on evoked objects is always nicest.

Would you be okay with a new method (`evoked.jointplot`) like this:

```
def jointplot(evoked, title=None, ts_args=None, topo_args=None):
"""Plots joint top and time series.

    evoked : evoked
        instance of evoked
    title : str
        The title of the figure
    ts_args : none | dict
        dict of args that are passed to evoked.plot
    topo_args : none | dict
        dict of args that are passed to evoked.plot_topo

    returns : fig"""
```

So if called without args, it'd just return default ts and topo plots. But you could still do e.g.

```
evoked.jointplot(title="P3 plot", ts_args=dict=(spatial_colors=True), topo_args=dict(times=.3))
```
and e.g. more elaborate args to topoplot.

**Comment by agramfort** [comment]
first let's make it work, test it on many datasets, then we'll make it nice
with a good API

**Comment by jona-sassenhagen** [comment]
Ok!

**Comment by jona-sassenhagen** [comment]
Closed by #2738
